<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="0">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>文本文件传输</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,2,3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="dat_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_5" rowstretch="2,1,3">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="title">
                  <string>码元速率</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_RB">
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="title">
                  <string>载波频率</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_Fc">
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_8">
                 <property name="title">
                  <string>采样频率</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_FS">
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QPushButton" name="pb_StartmoduAsk">
                  <property name="text">
                   <string>开始ASK调制</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_Savemodu">
                  <property name="text">
                   <string>保存ASK调制文件</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_StartmoduPsk">
                  <property name="text">
                   <string>开始PSK调制</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_SavemoduPsk">
                  <property name="text">
                   <string>保存PSK调制文件</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widget_3" native="true">
               <widget class="QPushButton" name="pb123456">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>20</y>
                  <width>75</width>
                  <height>23</height>
                 </rect>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>文本内容显示区</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QTextEdit" name="textEdit_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QTextEdit" name="textEdit_3"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>调制数据显示区</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>二进制数据显示区</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QChartView" name="widget2">
          <layout class="QGridLayout" name="gridLayout_7"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>音频文件传输</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="btnGetFile">
                 <property name="text">
                  <string>选择保存地址</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/images/images/104.bmp</normaloff>:/images/images/104.bmp</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="editOutputFile"/>
               </item>
               <item>
                <widget class="QLabel" name="LabPassTime">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>录制时间：</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,1">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QPushButton" name="pbStart">
                    <property name="text">
                     <string> 开始录音</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pbStop">
                    <property name="text">
                     <string>暂停录音</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pbCom">
                    <property name="text">
                     <string>停止录音</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>录音设置（QAudioEncoderSettings）</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="2" column="0">
                  <layout class="QFormLayout" name="formLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="maximumSize">
                      <size>
                       <width>85</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>音频Level:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <layout class="QVBoxLayout" name="levelsLayout"/>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pb_play">
                   <property name="text">
                    <string>播放录制文件</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_11">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>输入设备</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="comboDevices"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>音频编码</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="comboCodec"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_9">
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>采样率</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QComboBox" name="comboSampleRate"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>通道数</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QComboBox" name="comboChannels"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QChartView" name="chartView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTHX">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actiondat"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="actiondat_2"/>
    <addaction name="actionmodu"/>
    <addaction name="actionbao"/>
   </widget>
   <addaction name="menuTHX"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiondat">
   <property name="text">
    <string>打开文本文件</string>
   </property>
  </action>
  <action name="actiondat_2">
   <property name="text">
    <string>保存文本文件</string>
   </property>
  </action>
  <action name="actionmodu">
   <property name="text">
    <string>保存ASK调制文件</string>
   </property>
  </action>
  <action name="actionbao">
   <property name="text">
    <string>保存PSK调制文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
